Source/Controllers/SPAppController.m

```diff
@@ -1641,9 +1643,31 @@ - (IBAction)reloadBundles:(id)sender
 						}
 
 						if(!cmdData || readError) {
-							NSLog(@"“%@” file couldn't be read. (error=%@)", infoPath, readError);
-							NSBeep();
-							continue;
+							SPLog(@"“%@” file couldn't be read. (error=%@)", infoPath, readError.localizedDescription);
+							if(![alreadyBeeped objectForKey:bundle]){
+								NSBeep();
+								[alreadyBeeped setObject:@YES forKey:bundle];
+								// need to wait a few seconds for the window to load
+								delayCallback(^{
+									NSString *title = [NSString stringWithFormat:NSLocalizedString(@"Bundle '%@' is corrupt.", @"bundle corrupt message"), bundle];
+									NSString *message = [NSString stringWithFormat:NSLocalizedString(@"Would you like to delete bundle '%@'? This operation cannot be undone.", @"delete bundle informative message"), bundle];
+
+									[NSAlert createDefaultAlertWithTitle:title
+																 message:message
+													  primaryButtonTitle:NSLocalizedString(@"Delete", @"delete button")
+													primaryButtonHandler:^{
+
+										[self removeBundle:bundle];
+									}
+													 cancelButtonHandler:nil];
+
+								}, 4);
+								
+								continue;
+							}
+							else{
+								SPLog(@"already beeped for %@", bundle);
+							}
```

Comment:
If a bundle really is corrupt (which I think it is), should we prompt the user at all or should we just delete the bundle always? Then, if we deleted the bundle, there's a second choice to either notify or not notify the user. 

I think bundles got screwed up in an earlier update where I renamed two of them (it doesn't seem to cleanup right). I expect most users to hit this condition upon installing the next update.

Reply:
So remove this?

Reply:
I've deleted the bad bundles and kept and array of those deleted, in case we want to inform the user at a later date.

Reply:
Perfect!